graylog2-server/src/main/java/org/graylog2/configuration/PathConfiguration.java

```diff
@@ -17,11 +17,16 @@
 package org.graylog2.configuration;
 
 import com.github.joschi.jadconfig.Parameter;
+import com.github.joschi.jadconfig.ValidationException;
+import com.github.joschi.jadconfig.ValidatorMethod;
+import com.github.joschi.jadconfig.validators.PathExecutableValidator;
 
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
 public abstract class PathConfiguration {
+    public static final String CSV_FILE_LOOKUP_DIR = "csv_file_lookup_dir";
```

Comment:
How about using a more generic name? That way we can also use it for other data adapters that load files from the file system. (like the MaxMind adapter)

I would suggest `trusted_data_file_paths`. Let's use plural and allow a comma separated list of paths to be set. (we can use a `StringSetConverter` for the `@Parameter` to automatically set a set for the configured paths -- see `enabled_tls_protocols` option in `Configuration`)

Reply:
Thanks lot for the feedback @bernd! I like this. Will do.

Reply:
Sorry for nitpicking, but I'm not sure `trusted` is the right word here. 
There is nothing trustworthy about these paths themselves.
The facts are kind of backwards: We trust Graylog to access files in these paths.
How about `accessible_data_paths`?
Or more specifically `accessible_auxiliary_data_paths`?

Reply:
@mpfz0r The "trusted" naming was my suggestion and was the first that came to my mind. :smile: 

I agree, "trusted" could be misleading. I like your suggestions. :+1:

Reply:
@mpfz0r @bernd Thanks for the suggestions. I agree on "trusted" being possibly confusing. Good idea to find a better name.

"accessible" sounds too much like [accessibility](https://en.wikipedia.org/wiki/Computer_accessibility) to me, which is another topic all-together. 

What do you think about using "allowed" instead? I also think "allowed" is more correct for the specific effect of the setting (we're _allowing_ users to access them because of the setting). This is also consistent with allow list/block lists naming, which seems common.

I get the feeling that "auxiliary" might be too general, almost like it could be used for anything. "data_file" seemed a bit more specific (like it's intended to be used for things in Graylog that use files), but I am not sure about that. I am open to using auxiliary, but do you have any final thoughts or other suggestions for this part?

Reply:
`allowed` is fine with me. I agree that `auxiliary` is very generic. I've tried to distinguish that new setting from the `data_dir` setting.
We could call it `allowed_data_adapter_files` but I was hoping we could use that for other assets we get from the filesystem in the future..

Reply:
@mpfz0r Good point, putting "adapter" in there is too specific. We will want to use this for other purposes too. Perhaps "auxiliary is really the best choice. I'll go with that unless anyone has other ideas.